.github/workflows/ci_ckb_cli.yaml

```diff
@@ -0,0 +1,64 @@
+name: ci_ckb_cli
+on:
+  # Submit your review with a comment body containing "#ci_ckb_cli"
```

Comment:
Use the same condition as in `ci_integration_tests_ubuntu`.

Reply:
This CI should be triggered just like integration tests.

Reply:
add same prologue on ci_ckb_cli.yaml